module/core/src/Resources/config/services.yml

```diff
@@ -96,3 +96,15 @@ services:
 
     Ergonode\Core\Application\Serializer\SymfonySerializer:
         $serializer: '@serializer'
+
+    Ergonode\Core\Application\Serializer\Normalizer\AssocArrayDenormalizer:
```

Comment:
It seems to me that this should not be here because it generates an incorrect relationship between the modules.
It should be on the application level or in dedicated modules.